test/models/piece_test.rb

```diff
@@ -1,13 +1,16 @@
 require 'test_helper'
 
 class PieceTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
 
-  #test 'occupied' do
-  #	g = Game.create(row_position: x, col_position: y)
-  #	assert_equal (0, 0), g.piece.occupied?
-  	
-  #end 
+  test 'should have vertical forward obstruction' do
+    g = FactoryGirl.create(:game)
+    blk_king = FactoryGirl.create(:king, :game_id => g.id, :color => 'Black', :row_position => 0, :col_position => 4)
+    assert blk_king.is_obstructed?(3,4)
```

Comment:
be careful on this one.  the blk_king already exists. It doesn't matter in this test, but it might in future tests.  Can you use blk_king = g.pieces.where(...) ?